### PR TITLE
docs: migrate directives guide

### PR DESCRIPTION
**Pull request by bencodezen**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/main/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: aio/content/examples/built-in-directives/src/main.ts [hunk 1]
```diff
@@ -1,6 +1,7 @@
...
+import {bootstrapApplication, provideProtractorTestingSupport} from '@angular/platform-browser';
 
-import { AppModule } from './app/app.module';
+import {AppComponent} from './app/app.component';
 
-platformBrowserDynamic().bootstrapModule(AppModule)
-  .catch(err => console.error(err));
+bootstrapApplication(AppComponent, {
+  providers: [provideProtractorTestingSupport()],
+}).catch((err) => console.error(err));
```
Comment (MarkTechson) [inline aio/content/examples/built-in-directives/src/main.ts:7]: Let's drop the catch here.

As a side note, the `providers` COULD be listed in an `AppConfig` object, but I'm not hard on that one.

**Review by MarkTechson — CHANGES_REQUESTED** [review]
Just one slight nit, if we can change that this will be G2G!

**Review by twerske — APPROVED** [review]

**Review by MarkTechson — APPROVED** [review]
LGTM

File: aio/content/examples/structural-directives/src/app/hero.component.ts [hunk 2]
```diff
@@ -1,19 +1,24 @@
...
+import {Hero, heroes} from './hero';
 
 @Component({
+  standalone: true,
   selector: 'app-hero',
   template: `
     <button (click)="onLoadHero()">Load Hero</button>
     <p *appIfLoaded="heroLoadingState">{{ heroLoadingState.data | json }}</p>
   `,
+  imports: [CommonModule, IfLoadedDirective],
```
Comment (jessicajaniuk) [inline aio/content/examples/structural-directives/src/app/hero.component.ts:16]: Not blocking here, but we should switch these CommonModule imports to their more granular imports (NgIf, NgFor, etc)

File: aio/content/examples/built-in-directives/src/app/app.component.ts [hunk 3]
```diff
@@ -1,21 +1,45 @@
...
+  standalone: true,
+  // #enddocregion import-common-module, import-forms-module
   selector: 'app-root',
   templateUrl: './app.component.html',
-  styleUrls: ['./app.component.css']
+  styleUrls: ['./app.component.css'],
+
+  // #docregion import-common-module, import-forms-module
+  imports: [
+    CommonModule, // <-- import into the component
```
Comment (jessicajaniuk) [inline aio/content/examples/built-in-directives/src/app/app.component.ts:23]: We should be more granular where possible with CommonModule imports.

**Review by jessicajaniuk — APPROVED** [review]

**Comment by jessicajaniuk** [comment]
This PR was merged into the repository by commit f6fa69015296bd479481806dab91a6242f89c4e0.

**Pull request closed without merging**